Casks/firecamp.rb

```diff
@@ -1,16 +1,16 @@
 cask "firecamp" do
-  version "1.4.4"
-  sha256 "de6e433037328709b9c0d2e7b321550945cc323d92e8d7b83f5e7f3bd9c5a5aa"
+  version "2.0.1"
+  sha256 "f2567ee4184187473519fb8502a4d01b6cc265646247ffd4dfc80fa9347e917d"
 
-  url "https://firecamp.ams3.digitaloceanspaces.com/versions/mac/Firecamp-#{version}.dmg",
-      verified: "firecamp.ams3.digitaloceanspaces.com/"
+  url "https://firecamp.netlify.app/.netlify/functions/download?pt=mac",
+      verified: "firecamp.netlify.app/"
```

Comment:
This should use the versioned URL
```suggestion
  url "https://firecamp.ams3.digitaloceanspaces.com/versions/mac/Firecamp-#{version}.dmg",
      verified: "firecamp.ams3.digitaloceanspaces.com/"
```